test/TestExtensions/OrleansTestingBase.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using Orleans;
+using Orleans.Runtime;
+
+namespace UnitTests.Tester
+{
+    public abstract class OrleansTestingBase
+    {
+        protected static readonly Random random = new Random();
+
+        public Logger logger => GrainClient.Logger;
+
+        protected static IGrainFactory GrainFactory => GrainClient.GrainFactory;
+
+        protected static long GetRandomGrainId()
+        {
+            return random.Next();
+        }
+    }
+}
```

Comment:
why do you removed a lot of comment and method `TestSilosStarted` from this class when you moved it to TestExtensions?